src/CosmosEditorManager.ts

```diff
@@ -76,7 +78,8 @@ export class CosmosEditorManager {
     }
 
     private async updateEditor(data: {}, textEditor: vscode.TextEditor): Promise<void> {
```

Comment:
data: T?

Reply:
I don't see an easy way to get the type from the ICosmosEditor generic